buildout.cfg

```diff
@@ -53,7 +53,7 @@ zope.testrunner = 4.7.0
 [script]
 recipe = zc.recipe.egg
 dependent-scripts = true
-interpreter = python
+interpreter = py
```

Comment:
Why's this? 

(My guess: you use a virtualenv to get rid of the bootstrap.py? The change sounds OK then.)